changelog.json

```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "1.5.0-beta2": [
+      "[New] Successful merge banner - #5851",
+      "[New] Merge button in branch selector - #5929. Thanks @bruncun!",
+      "[New] Merge conflict resolution flow - #5400",
```

Comment:
```suggestion
      "[New] Improved flow for resolving merge conflicts - #5400",
```

Reply:
I think the word `improved` is confusing here. I know we handled conflicts in a manner before so maybe we could say "[New] Merge conflict resolution helper - #5400" instead? or something like that? cc @ampinsk

Reply:
I like @billygriffin's suggestion here https://github.com/desktop/desktop/pull/6040#discussion_r229402349

Reply:
I like what @billygriffin suggested here!